service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsController.java

```diff
@@ -148,9 +159,16 @@ private void removeDocumentFromData(Map<String, Object> data) {
         List<Element<C2DocumentBundle>> c2DocumentBundle = defaultIfNull(caseData.getC2DocumentBundle(),
             Lists.newArrayList());
 
+        List<SupportingEvidenceBundle> updatedSupportingEvidenceBundle =
+            unwrapElements(caseData.getTemporaryC2Document().getSupportingEvidenceBundle())
+                .stream()
+                .map(supportingEvidence -> supportingEvidence.toBuilder().dateTimeUploaded(LocalDateTime.now()).build())
```

Comment:
How this is tested? That appropriate date is added to the document? 

I'ts not much logic above, but this probably should live in the service.

Reply:
I hope you know how much i wanted to move this to service 😄 , there wasn't a service at all when i was working on this... now i added it to do validation. I don't want to put just my code in this PR to service which is bad(i think), so i have to move to all private method which would mean adding tests to things not related to story. If i get time, I would love to do it. Don't want to hold up @Jamesnelson117 . I promise, If i don't do this before this PR is merged, I will do follow up PR 😄 .

Reply:
No that's fine, we can do it later! Feel free to create the ticket please so we don't forget :).